lib/rucio/daemons/conveyor/common.py

```diff
@@ -305,23 +309,37 @@ def bulk_group_transfers_for_fts(transfers, policy='rule', group_bulk=200, sourc
                   'selection_strategy': source_strategy if source_strategy else activity_source_strategy.get(str(transfer['file_metadata']['activity']), default_source_strategy),
                   'request_type': transfer['file_metadata'].get('request_type', None),
                   'activity': str(transfer['file_metadata']['activity'])}
-
         if verify_checksum != 'none':
             set_checksum_value(t_file, checksums_to_use)
-
         multihop = transfer.get('multihop', False)
         strict_copy = transfer.get('strict_copy', False)
         use_ipv4 = transfer.get('use_ipv4', False)
 
+        if multi_vo:
+            vo = transfer['account'].vo
+
         external_host = transfer['external_host']
         activity = t_file['activity']
 
-        if external_host not in grouped_transfers:
-            grouped_transfers[external_host] = {}
-            grouped_jobs[external_host] = []
+        if multi_vo:
```

Comment:
I have a feeling that none of the changes in bulk_group_transfers bellow this line are needed. That adding 
```
            if vo:
                job_key = job_key + ',vo=%s' % vo 
````
after https://github.com/rucio/rucio/blob/e6940d402b168b601b92c7b3996bc7446d6d4cae/lib/rucio/daemons/conveyor/common.py#L390
is enough. Did you try it this way ?

Reply:
never mind, I'm wrong